src/lib/y2storage/planned/partition.rb

```diff
@@ -60,6 +60,9 @@ class Partition < Device
       #   flag but is not needed in our grub2 setup.
       attr_accessor :bootable
 
+      # @return [String] name of the LVM volume group where the partition belongs
+      attr_accessor :lvm_volume_group_name
```

Comment:
I guess this should, somehow, replace the existing `lvm_pv?` and its usage during the proposal.